src/service/resources-impl.js

```diff
@@ -1585,6 +1593,26 @@ export class Resources {
     }
   }
 
+  /**
+   * Returns true if reflow will not be caused by expanding the given element's
+   * width to the given amount.
+   * @param {!Element} element
+   * @param {number} width
+   * @return {boolean}
+   */
+  isWidthOnlyReflowFreeExpansion_(element, width) {
+    const parent = element && element.parentElement;
+    // If the element has siblings, it's possible that a width-expansion will
+    // cause some of them to be pushed down.
+    if (!parent || parent.childElementCount > 1) {
+      return false;
+    }
+    const parentWidth = parseInt(computedStyle(this.win, parent).width, 10);
```

Comment:
This causes a measure in the middle of a mutation phase.

Reply:
@jridgewell Is there a way of getting the width of this element without causing a measure? If not, I guess I can only return true here if the parent has a fixed layout (in which case I can just grab the width attribute).

Reply:
If it's an amp-element, calling `implementation.getLayoutWidth` would work. For regular elements, there's no way to do it.

Reply:
Removed call to getComputedStyle.